account_default_draft_move/invoice_view.xml

```diff
@@ -22,7 +22,9 @@
       <field name="type">form</field>
       <field name="arch" type="xml">
         <xpath expr="//button[@name='invoice_cancel']" position="after">
-          <button name="invoice_cancel" states="open" string="Cancel Invoice" groups="account.group_account_invoice"/>
+          <field name="update_posted" invisible="1"/>
+          <button name="invoice_cancel" states="open" string="Cancel Invoice" groups="account.group_account_invoice"
+                  attrs="{'invisible': ['|', ('update_posted', '=', False)]}"/>
```

Comment:
Hi @laetitia-gangloff ! Thanks a lot ! Indeed there is something unusual going on here.

There is no invisible attribute in the parent form. 

I just made a quick test on runbot.

With this version, we have the cancel button twice when account_cancel is installed: 
![image](https://cloud.githubusercontent.com/assets/692075/26091737/5d2e8d7a-3a0d-11e7-8bb7-80502c5751dd.png), which totally makes sense since the inherited view adds a second button (with the same name).

WIth the original code (with the `'|'`), we have only one button.

So I revert for know... there is something I don't understand going on here.